docs/book/src/developers/releasing.md

```diff
@@ -1,18 +1,40 @@
-# Release Process
+# CAPZ Releases
 
-## Update metadata.yaml (skip for patch releases)
+## Release Cadence
+
+CAPZ minor versions (that is, 1.**5**.0 versus 1.**4**.x) are released every two months.
+
+CAPZ patch versions (for example, 1.5.**2** versus 1.5.**1**) are released every two weeks. A patch release may bypass this cadence if circumstances warrant.
```

Comment:
We agreed during a recent office hours that we would aim for a weekly release "check in". In other words, every week we will poll the community for consensus around whether or not we should cut a patch given what's in the release branches. So if we have a regular routine of cherry-picking into the release branch w/ community support, we will in effect have a weekly release cadence. Or if the community prefers less frequency, it will be less frequent.

2022 August 25th office hours discussion notes:

https://docs.google.com/document/d/1P2FrRjuCZjGy0Yh72lwWCwmXekSEkqliUVTmJy_ETIk